detectron2/data/datasets/coco.py

```diff
@@ -280,6 +286,131 @@ def file2id(folder_path, file_path):
     return dataset_dicts
 
 
+def convert_to_coco_dict(dataset_name):
+    """
+    Convert a generic dataset into COCO json format
+
+    Generic dataset description can be found here:
+    https://detectron2.readthedocs.io/tutorials/datasets.html#register-a-dataset
+
+    COCO data format description can be found here:
+    http://cocodataset.org/#format-data
+
+    Args:
+        dataset_name: name of the source dataset
```

Comment:
Add "must be registered in `DatastCatalog` and in detectron2's standard format."
same for the other function below

Reply:
done